src/ape/managers/config.py

```diff
@@ -47,9 +44,20 @@ class ConfigManager:
     contracts_folder: Path = None  # type: ignore
     dependencies: Dict[str, str] = {}
     deployments: Dict[str, Dict[str, List[DeploymentConfig]]] = {}
-    plugin_manager: PluginManager
+    plugin_manager: ClassVar[PluginManager] = injected_before_use()  # type: ignore
     _plugin_configs_by_project: Dict[str, Dict[str, ConfigItem]] = {}
 
+    def __init__(
+        self,
+        *,
```

Comment:
Why is this here?

Reply:
The * ensures that all params passed after it are done by keyword and not position.

Reply:
ah okay. but won't this ignore any args provided?

Reply:
It will get very angry about the args, do we want to support *arg, **kwargs on these init methods?

Reply:
No, I would just remove it if that's the case